package.json

```diff
@@ -8,6 +8,7 @@
     "url": "https://github.com/microsoft/fluentui"
   },
   "scripts": {
+    "storybook": "start-storybook -p 6006 --no-manager-cache",
```

Comment:
There was a suggestions to use "--no-manager-cache" when using the Themes to avoid caching between unrelated instances, which helped me with the theme testing locally but this syntax actually should be reverted from this PR.